great_expectations/expectations/core/expect_column_distinct_values_to_be_in_set.py

```diff
@@ -45,39 +45,9 @@ class ExpectColumnDistinctValuesToBeInSet(ColumnAggregateExpectation):
     expect_column_distinct_values_to_be_in_set is a \
     [Column Aggregate Expectation](https://docs.greatexpectations.io/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_column_aggregate_expectations).
 
-    The success value for this expectation will match that of \
-    [expect_column_values_to_be_in_set](https://greatexpectations.io/expectations/expect_column_values_to_be_in_set).
-
-    For example:
-    ::
-
-        # my_df.my_col = [1,2,2,3,3,3]
-        >>> my_df.expect_column_distinct_values_to_be_in_set(
-                "my_col",
-                [2, 3, 4]
-            )
-        {
-            "success": false
-            "result": {
-                "observed_value": [1,2,3],
-                "details": {
-                    "value_counts": [
-                        {
-                            "value": 1,
-                            "count": 1
-                        },
-                        {
-                            "value": 2,
-                            "count": 1
-                        },
-                        {
-                            "value": 3,
-                            "count": 1
-                        }
-                    ]
-                }
-            }
-        }
+    Column Aggregate Expectations are one of the most common types of Expectation.
+    They are evaluated for a single column, and produce an aggregate Metric, such as a mean, standard deviation, number of unique values, column type, etc.
+    If that Metric meets the conditions you set, the Expectation considers that data valid.
 
     Args:
         column (str): \
```

Comment:
General question: What purpose do the backslashes `\` in the docstrings serve?

Reply:
I'm not sure -- those are inherited from when these docstrings were initially formatted re: google styling. I've left them in place wherever they occurred.